docs/userguide/dockervolumes.md

```diff
@@ -59,6 +59,12 @@ This will create a new volume inside a container at `/webapp`.
 > You can also use the `VOLUME` instruction in a `Dockerfile` to add one or
 > more new volumes to any container created from that image.
 
+Docker defaults to a read-write volume but we can also mount a directory
+read-only.
+
+    $ docker run -d -P --name web -v /opt/webapp:ro training/webapp python app.py
+
+
```

Comment:
perhaps `Docker volumes default to mount in read-write mode, but you can also set it to be mounted read-only.`